project-icp/bee-pollinator-app
# Remove tiler, layer control, layer settings, and related Ansible & deployment code

This PR removes code related to the tiler, layer control, and layer settings from the app and deployment code. In addition to dropping the tiler VM and the related Ansible and Vagrant settings, it also cuts the layer settings file and removes references to the layer settings and `DRB_PERIMETER` constant from Django, drops the custom layer control and the code to create the layers from the frontend, and cuts all the observations code including stuff on the frontend and in Nginx

With the change, we've dropped the basemap, too, and we'll add it back using a standard leaflet layers control:

<img width="953" alt="screen shot 2016-10-17 at 11 12 09 am" src="https://cloud.githubusercontent.com/assets/4165523/19443541/0fd2df70-945b-11e6-8c56-660b8a8d7652.png">

**Testing**
- get this branch, then `vagrant up --provision app`, and once that's done run `./scripts/bundle.sh`
- visit `localhost:8000` and open the browser console
- verify that you see the above screen (which doesn't include the layers control), and that there aren't any errors in the console.

Note that the "Loading..." dropdown is for the boundary layer selector which will be removed in #3 

Connects #4
